### DISCUSSION TITLE
Make installing directory optional

### DISCUSSION THREAD

**Discussion by Tenkotataii** [ppy/osu#24649] [Ideas] [answered]
I had a bad time while playing osu!lazer and seeing it being downloaded on my nearly full C: drive. I think being able to decide which disk to use would help me and all other people who use osu!lazer.

**Comment by peppy** [comment, marked as answer]
You can change the install path from settings!

**Reply by Tenkotataii** [reply]
why don't you make an installer like you did on stable?

**Reply by peppy** [reply]
Might happen eventually. But it's a lot more effort to get right, and cross-platform support is hard.
